builtin/logical/pki/chain_test.go

```diff
@@ -151,6 +153,14 @@ func (c CBGenerateIntermediate) Run(t testing.TB, b *backend, s logical.Storage,
 	if len(c.CommonName) > 0 {
 		data["common_name"] = c.CommonName
 	}
+	if len(c.SKID) > 0 {
+		// Copy the SKID from an existing, already-issued cert.
+		otherPEM := knownCerts[c.SKID]
+		otherCert := ToCertificate(t, otherPEM)
+
+		data["skid"] = hex.EncodeToString(otherCert.SubjectKeyId)
```

Comment:
Realized only here that we can't pass in an existing cert to cross-sign.  We might want to add that?

Nice tests though

Reply:
I'm mostly against it. It is definitely a lot of work (in the existing signing framework via certutil) to add deserialization of existing certificates back into the input parameters format... 

It definitely requires a bit of careful though and structuring...

Reply:
(I have gone ahead and added a separate test to verify the copy, and I did so here as well).